### DISCUSSION TITLE
E: Unsupported file ./jamulus_headless_3.9.0_ubuntu_amd64.deb given on commandline

### DISCUSSION THREAD

**Discussion by ve3meo** [jamulussoftware/jamulus#2784] [Technical Support & Help] [answered]
Cannot upgrade server to 3.9.0 due to above error. Ubuntu 22.04.1 LTS virtual machine currently running 3.8.1. Any tips for this lost wanderer in the Linux woods? (ignore the strikethroughs - I don't know why they appear)

root@ganaraskals:~# add-apt-repository universe
Adding component(s) 'universe' to all repositories.
Press [ENTER] to continue or Ctrl-c to cancel.
Hit:1 http://security.ubuntu.com/ubuntu jammy-security InRelease
Hit:2 http://archive.ubuntu.com/ubuntu jammy InRelease
Hit:3 http://archive.ubuntu.com/ubuntu jammy-updates InRelease
Hit:4 http://archive.ubuntu.com/ubuntu jammy-backports InRelease
Reading package lists... Done
root@ganaraskals:~# apt-get update
Hit:1 http://security.ubuntu.com/ubuntu jammy-security InRelease
Hit:2 http://archive.ubuntu.com/ubuntu jammy InRelease
Hit:3 http://archive.ubuntu.com/ubuntu jammy-updates InRelease
Hit:4 http://archive.ubuntu.com/ubuntu jammy-backports InRelease
Reading package lists... Done
root@ganaraskals:~# apt update
Hit:1 http://archive.ubuntu.com/ubuntu jammy InRelease
Hit:2 http://archive.ubuntu.com/ubuntu jammy-updates InRelease
Hit:3 http://archive.ubuntu.com/ubuntu jammy-backports InRelease
Hit:4 http://security.ubuntu.com/ubuntu jammy-security InRelease
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
All packages are up to date.
root@ganaraskals:~# apt install ./jamulus_headless_3.9.0_ubuntu_amd64.deb
Reading package lists... Done
E: Unsupported file ./jamulus_headless_3.9.0_ubuntu_amd64.deb given on commandline
root@ganaraskals:~#

**Comment by jujudusud** [comment, marked as answer]
go to the package place on your disk and use:
`apt install jamulus_headless_3.9.0_ubuntu_amd64.deb`

Is it ok ?

**Reply by ve3meo** [reply]
That did the trick, thanks. But what a misleading error statement! If the file could not be found, then I would expect something like "E: Unable to locate package ..."

**Reply by hoffie** [reply]
I'm actually surprised that the `./` syntax did not work, but the path-less syntax did. I've just tried this on Ubuntu 18.04/20.04/22.04 and  the path-based syntax has always worked. I'm wondering what was different here...

@gilgongo did tests with similar results I think.

**Reply by ve3meo** [reply]
The error was mine. I did not realise that I had uploaded the package to a different folder than where my ssh terminal was. I'm sure the path-based syntax would have worked had this Linux dilettante known what he was doing!

**Reply by gilgongo** [reply]
It seems to be inconsistent for me. I've had it work with `./` and not work with pathless syntax ("E: Unable to locate package")  sometimes I think but I'm not sure why.

**Reply by hoffie** [reply]
> The error was mine. I did not realise that I had uploaded the package to a different folder than where my ssh terminal was. I'm sure the path-based syntax would have worked

Ah, great, thanks for the feedback!
